src/script/src/manager.rs

```diff
@@ -112,19 +128,43 @@ impl ScriptManager {
             .context(CompilePythonSnafu { name })
     }
 
+    /// Get or create the scripts table in the catalog
+    pub fn get_scripts_table(&self, catalog: &str) -> Option<ScriptsTableRef<E>> {
+        self.tables.read().unwrap().get(catalog).cloned()
+    }
```

Comment:
this function only try to get the table, but won't create one if not exist

Reply:
Sorry, the doc is misunderstood.